.github/workflows/helm.yaml

```diff
@@ -39,6 +43,35 @@ jobs:
           if [[ -n "$changed" ]]; then
             echo "::set-output name=changed::true"
           fi
+      
+      # Force linting/test in debug runs with a change (this action only runs on chart and helm.yaml workflow diffs anyways)
+      - name: Change Kind to deployment
+        run: "sed -i \"s|kind: [^ ]*|kind: Deployment|g\" charts/descheduler/values.yaml"
```

Comment:
this seems odd to do, when we can have a specific values file that can do this.

Reply:
This was only to force ct lint and ct install to show changes, it is unrelated to this value change itself. I want to remove this when the work is finished.

Reply:
Initially I was planning on leaving this, but now I see it is too ugly/confusing :sweat_smile: